packages/client-beta/src/apps/Player/components/YouTubePlayer/YouTubePlayer.tsx

```diff
@@ -0,0 +1,21 @@
+import { onMount, onCleanup } from 'solid-js';
+import { YoutubePlayerService } from '../../services/youtube-player-service';
+import { SocketConnectionClient } from '../../../../services/socket-connection-client';
+
+const YOUTUBE_PLAYER_DOM_ID = 'youtube-player';
+
+function YouTubePlayer() {
+  onMount(() => {
+    SocketConnectionClient.initializeConnection();
+    YoutubePlayerService.initialize(YOUTUBE_PLAYER_DOM_ID);
+  });
+
+  onCleanup(() => {
+    SocketConnectionClient.disconnect();
+    YoutubePlayerService.cleanup();
+  });
+
+  return <div id={YOUTUBE_PLAYER_DOM_ID} />;
```

Comment:
Nie możemy tu skorzystać z refa?

Reply:
W sumie to rzeczywiście możemy, ale i tak to id wykorzystuje, bo muszę nadpisać style dla tego iframe, który pojawia się zamiast tego elementu. Te style dodaje, bo inaczej mi nie chciał player reagować wielkością na resize okienka

Reply:
W sumie to rzeczywiście możemy, ale i tak to id wykorzystuje, bo muszę nadpisać style dla tego iframe, który pojawia się zamiast tego elementu. Te style dodaje, bo inaczej mi nie chciał player reagować wielkością na resize okienka